package.json

```diff
@@ -90,5 +94,10 @@
         "handlebars": "^4.7.8",
         "parse5": "^7.1.2",
         "wanakana": "^5.3.1"
+    },
+    "lint-staged": {
+        "*.js": "eslint --cache --fix",
```

Comment:
I think it's better to be specific on the sequence of commands, instead of having two different rules (`*.js`, `*.{js,css,md}`) handling the same file

So like
```json
"*.js": [
  "prettier --write",
  "eslint --cache --fix"
],
```

instead of 

```json
"*.js": "eslint --cache --fix",
"*.{js,css,md}": "prettier --write"
```

---

I've tried running `npx lint-staged -d` with this commit and got an output such as

```
[STARTED] *.js — 1 file
[STARTED] *.css — 0 files
[STARTED] *.{js,css,md} — 1 file
[SKIPPED] *.css — no files
[STARTED] eslint --cache --fix
[STARTED] prettier --write
[COMPLETED] prettier --write
[COMPLETED] *.{js,css,md} — 1 file
[COMPLETED] eslint --cache --fix
```

which shows that `eslint --cache --fix` and `prettier --write` has a race condition

---

After my suggested change it'll be

```
[STARTED] *.js — 1 file
[STARTED] *.css — 0 files
[STARTED] *.{css,md} — 0 files
[SKIPPED] *.css — no files
[SKIPPED] *.{css,md} — no files
[STARTED] prettier --write
[COMPLETED] prettier --write
[STARTED] eslint --cache --fix
[COMPLETED] eslint --cache --fix
[COMPLETED] *.js — 1 file
```
which is clear that `eslint` runs after `prettier` instead of running them concurrently

Reply:
Thanks for testing, I was careless and didn't fully check how this affected js files. I removed the prettier action from css and js for now so only markdown will be affected by prettier, hopefully that can be changed in the future to make everything consistent.